src/Workspaces/Core/Portable/Recommendations/AbstractRecommendationService.cs

```diff
@@ -88,7 +88,20 @@ internal bool ShouldIncludeSymbol(ISymbol symbol)
 
                 if (_context.IsEnumTypeMemberAccessContext)
                 {
-                    return symbol.Kind == SymbolKind.Field;
+                    // If you have a field and an enum with the same name,
+                    // we want to show instance members (through the field)
+                    // so you can call CompareTo, etc, in addition to the 
+                    // enum fields. However, we don't want to show statics
+                    // from the Enum base type.
+                    if (symbol.Kind == SymbolKind.Field)
```

Comment:
This seems weird.  Presumably properties are also affected.  (And locals/parameters if they happened to be the same casing as the enum?)